refgenconf/const.py

```diff
@@ -94,6 +94,18 @@
     "CFG_ASSET_CHILDREN_KEY", "CFG_TAG_DESC_KEY", "CFG_ASSET_CHECKSUM_KEY", "CFG_ASSET_TAGS_KEY",
     "CFG_ASSET_RELATIVES_KEYS", "CFG_ARCHIVE_CONFIG_KEY", "CFG_ARCHIVE_KEY_OLD"]
 
+# hook identifiers, A_* (after/post) B_* (before/pre)
+B_UPDATE_HOOK = "pre_update"
+A_UPDATE_HOOK = "post_update"
+B_PULL_HOOK = "pre_pull"
+A_PULL_HOOK = "post_pull"
+B_TAG_HOOK = "pre_tag"
+A_TAG_HOOK = "post_tag"
+B_LIST_HOOK = "pre_list"
+A_LIST_HOOK = "post_list"
```

Comment:
instead of A_* and B_*, which just adds unnecessary cognitive load, why not just use "pre" and "post"? I guess it would be easier to say:

PRE_UPDATE_HOOK = "pre_update"

But then I always wonder, when you're just defining the name of the constant to be exactly the same as the constant, is this *really* that useful of a construct?... I suppose!

Reply:
just wanted to keep the const name short and use one letter encoding. I'm fine with switching to whatever, if you think it adds complexity. 

the only gain that I'm getting at with these is not to redefine keys in multiple places and therefore avoiding possible errors caused by typos

Reply:
im fine with the consts but I don't like A_ and B_ over PRE_ and POST_.

Reply:
you could put these in a dict called "HOOKS" and refer to them as `HOOKS[PRE_TAG]` and `HOOKS[POST_PULL]`